x-pack/test/apm_api_integration/tests/alerts/helpers/get_connector_action_message.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Client } from '@elastic/elasticsearch';
+
+interface ActionMessageAggResponse {
+  message: {
+    buckets: Array<{
+      key: string;
+      doc_count: number;
+    }>;
+  };
+}
+
+export async function getConnectorActionMessage({
+  messageId,
+  esClient,
+  index,
+}: {
+  messageId?: string;
+  esClient: Client;
+  index: string;
+}): Promise<string> {
+  const searchParams = {
+    index,
+    body: {
+      fields: ['id.keyword', 'message.keyword'],
+      aggs: {
+        id: {
+          filter: {
+            term: {
+              'id.keyword': messageId,
+            },
+          },
+          aggs: {
+            message: {
+              terms: {
+                field: 'message.keyword',
```

Comment:
You only want the first, right?

```suggestion
                field: 'message.keyword',
                size: 1
```